features/docs/gegeven-stap-definities-persoon.feature

```diff
@@ -56,6 +56,20 @@ Functionaliteit: Persoon, Inschrijving gegeven stap definities
       | op een onbekende datum | het 'Verenigde Arabische Republiek' | man           |         00000000 |              9047 | M                   |         | buitenlandse geboorteakte |
       | in februari 2021       | 'België'                            |               |         20210200 |              5010 |                     |         | buitenlandse geboorteakte |
 
+  @integratie
+  Scenario: de {datum} in {land omschrijving} geboren {geslacht type} '{persoon aanduiding}' zonder burgerservicenummer
```

Comment:
Misschien mis ik iets, maar ik zie niet direct waarvoor we een persoon moeten opvoeren waarbij geboorteland opgegeven moet worden.

Voor een minderjarige persoon is geboorteland relevant, maar dan moet er altijd een burgerservicenummer zijn.
Voor een ouder, partner of kind kan er geen burgerservicenummer zijn, maar is geboorteland niet relevant.

Voor een ouder of partner (met of zonder burgerservicenummer) is alleen relevant of de persoon meerderjarig is.

Ik denk dan aan een stap `Gegeven de meerderjarige persoon {string} zonder burgerservicenummer`

Reply:
aangepast